src/javascript/SelectorTypes/Picker/accordionItems/accordionItems.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {Constants} from '~/SelectorTypes/Picker/Picker2.constants';
+import {renderer} from '~/SelectorTypes/Picker/accordionItems/renderer';
+import {Collections} from '@jahia/moonstone';
+
+const TARGETS = {
+    default: `${Constants.ACCORDION_ITEM_TYPES.PAGES}-${Constants.ACCORDION_ITEM_TYPES.CONTENT_FOLDERS}-${Constants.ACCORDION_ITEM_TYPES.MEDIA}`,
+    editorial: `${Constants.ACCORDION_ITEM_TYPES.PAGES}-${Constants.ACCORDION_ITEM_TYPES.CONTENT_FOLDERS}`,
+    editoriallink: `${Constants.ACCORDION_ITEM_TYPES.PAGES}-${Constants.ACCORDION_ITEM_TYPES.CONTENT_FOLDERS}`,
+    image: `${Constants.ACCORDION_ITEM_TYPES.MEDIA}`,
+    file: `${Constants.ACCORDION_ITEM_TYPES.MEDIA}`,
+    page: `${Constants.ACCORDION_ITEM_TYPES.PAGES}`,
+    folder: `${Constants.ACCORDION_ITEM_TYPES.MEDIA}`,
+    contentfolder: `${Constants.ACCORDION_ITEM_TYPES.CONTENT_FOLDERS}`,
+    category: `${Constants.ACCORDION_ITEM_TYPES.CATEGORY}`
+};
+
+export const getItemTarget = pickerType => {
+    const target = TARGETS[pickerType];
+    return target ? target : TARGETS.default;
+};
+
+export const getAccordionItemsOverrideObject = () => ({
+    [Constants.ACCORDION_ITEM_TYPES.PAGES]: {...renderer, requiredSitePermission: undefined},
+    [Constants.ACCORDION_ITEM_TYPES.CATEGORY]: {...renderer, requiredSitePermission: undefined},
+    [Constants.ACCORDION_ITEM_TYPES.MEDIA]: {...renderer, requiredSitePermission: undefined}
+});
+
+export const registerAccordionItems = registry => {
+    // These are jcontent accordion items, additional targets are added to enhance selection
+    const pagesItem = registry.get(Constants.ACCORDION_ITEM_NAME, Constants.ACCORDION_ITEM_TYPES.PAGES);
+    const contentFoldersItem = registry.get(Constants.ACCORDION_ITEM_NAME, Constants.ACCORDION_ITEM_TYPES.CONTENT_FOLDERS);
+    const mediaItem = registry.get(Constants.ACCORDION_ITEM_NAME, Constants.ACCORDION_ITEM_TYPES.MEDIA);
+
+    if (pagesItem) {
+        pagesItem.targets.push(
+            {id: TARGETS.default, priority: '50'},
```

Comment:
Sorry, maybe it was easier / more clear to redefine new items but just inherits from the jcontent ones. Doing that we would not need the `accordionItemProps`, and code would be more readable :
```
registry.add('accordionItem', 'pickerPages', pagesItem, renderer, {
   targets: ['default:50', 'editorial:50', 'page:50']
});
```
what do you think ?

Reply:
I think it's cleaner than passing render override via props.